File: 백준/Bronze/34552. 디딤돌 장학금/디딤돌 장학금.c
#include <stdio.h>
int main(void) {
    int n,b,s,rlt=0,m[11];
    double l;
    for(int i=0;i<11;i++)
        scanf("%d",&m[i]);
    scanf("%d",&n);
    for(int i=0;i<n;i++) {
        scanf("%d %lf %d",&b,&l,&s);
        if(l>=2.0 && s>=17)
            rlt+=m[b];
    }
    printf("%d",rlt);
    return 0;
}

File: 백준/Bronze/34552. 디딤돌 장학금/디딤돌 장학금.py
import sys
input=lambda:sys.stdin.readline().rstrip()
m,rlt=[*map(int,input().split())],0
for i in range(int(input())):
    b,l,s=map(float,input().split())
    if l>=2.0 and s>=17:
        rlt+=m[int(b)]
print(rlt)